test/node-api/test_buffer/test-external-buffer.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const common = require('../../common');
+const binding = require(`./build/${common.buildType}/test_buffer`);
+const assert = require('assert');
+
+// Regresion test for https://github.com/nodejs/node/issues/31134:
```

Comment:
```suggestion
// Regresion test for https://github.com/nodejs/node/issues/31134 :
```

Reply:
I’ll go with, thanks :)

```suggestion
// Regression test for https://github.com/nodejs/node/issues/31134
```